simpleflow/activity.py

```diff
@@ -87,7 +87,25 @@ def name(self):
         return '.'.join([prefix, name])
 
     def __repr__(self):
-        return 'Activity(name={}, version={}, task_list={})'.format(
+        return 'ActivityType(name={}, version={}, task_list={})'.format(
             self.name,
             self.version,
             self.task_list)
+
+
+class Activity(object):
```

Comment:
After seeing the PR, I wonder if "Activity" is really needed, isn't it the same as "ActivityTask" ?